Mayhemfile

```diff
@@ -0,0 +1,20 @@
+# Namespaced project name that the target belongs to
+project: subtle
+
+# Target name (should be unique within the project)
+target: subtle
```

Comment:
As with other repos, please:
1. Comment or remove the `image` field
2. Specify a single command in the Mayhemfile (or create multiple Mayhemfiles for each target)
3. Ensure that the `target` field reflects the target binary being run